app/app.js

```diff
@@ -6,10 +6,15 @@ import { appReady, appUnload } from 'pixelhandler-blog/utils/metrics';
 
 Ember.MODEL_FACTORY_INJECTIONS = true;
 
-window.showdown = new Showdown.converter();
+debugger;
```

Comment:
Wip I wonder why the showdown lib is not loaded with app.import in the Brocfile.js build